data/reference/syntax.adoc

```diff
@@ -137,138 +164,88 @@ literal
 open
 --
 
-[pass]
-++++
-passthrough
-++++
-
-[quote,attribution,citation title and information]
+[,attribution,citetitle]
```

Comment:
I didn't know this syntax, it might just be me but I prefer the explicit version 😄

Reply:
The style name is always optional when it matches the default style name for a delimited block. I'd like to leave it this way since I don't want to give the impression that the quote style is required (even when the attribution is not used).